engine/src/test/scala/cromwell/SimpleWorkflowActorSpec.scala

```diff
@@ -111,7 +111,7 @@ class SimpleWorkflowActorSpec extends CromwellTestkitSpec with BeforeAndAfter {
     }
 
     "fail when a call fails" in {
-      val expectedError = "Call goodbye.goodbye: return code was 1"
+      val expectedError = "Call goodbye.goodbye:-1:1: return code was 1"
```

Comment:
This brings to the fore that `JobKey#toString` is kind of lame, and this test is dependent on its format.  🙁